content/applications/productivity/calendar/google.rst

```diff
@@ -109,27 +112,43 @@ next to :guilabel:`Google Calendar`.
    :align: center
    :alt: The Google Calendar checkbox in General Settings.
 
-Next, copy and paste the Client ID and the Client Secret from the Google Calendar API Credentials
-page into their respective fields below the :guilabel:`Google Calendar` checkbox. Then, click
-:guilabel:`Save`.
+Next, copy and paste the *Client ID* and the *Client Secret* from the Google Calendar API
+credentials page into their respective fields below the :guilabel:`Google Calendar` checkbox. Then,
+click :guilabel:`Save`.
 
-Finally, open the Calendar module in Odoo and click on the :guilabel:`Google` sync button to sync
-Google Calendar with Odoo.
+Sync calendar in Odoo
+=====================
+
+Finally, open the :menuselection:`Calendar` app in Odoo and click on the :guilabel:`Google` sync
+button to sync Google Calendar with Odoo.
 
 .. image:: google/sync-google.png
```

Comment:
Is this screenshot only for 16? @tiku-odoo

Reply:
This should be relevant for all versions @StraubCreative